lib/event-bus.js

```diff
@@ -16,11 +16,14 @@ module.exports = class EventBus {
         return;
       }
 
-      const evnt = JSON.parse(newValue);
+      const evnt = newValue && JSON.parse(newValue);
```

Comment:
On second look at the changes here in `event-bus.js`, this needs a separate PR. Pain the rear, I know. But it hasn't been discussed and it's out of scope for the SSL fix.